### PR TITLE
Mark Xcode archives generated by gym with extended attribute

### PR DESCRIPTION
**Pull request by mpirri**

### CODE REVIEW

File: lib/gym/runner.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 require 'pty'
 require 'open3'
 require 'fileutils'
+require 'shellwords'
```
Comment (i2amsam) [inline lib/gym/runner.rb]: Does this need to be added to the gemspec?

File: lib/gym/runner.rb [hunk 2]
```diff
@@ -84,6 +90,7 @@ def build_app
 
       UI.success "Successfully stored the archive. You can find it in the Xcode Organizer."
       UI.verbose("Stored the archive in: " + BuildCommandGenerator.archive_path)
+      mark_archive_as_built_by_gym(BuildCommandGenerator.archive_path)
```
Comment (i2amsam) [inline lib/gym/runner.rb]: Should this happen before the success message?

File: lib/gym/runner.rb (hunk 1 above)
Comment (mpirri) [inline lib/gym/runner.rb]: Hmmm, excellent question.  Other fastlane tools (fastlane, sign, snapshot) have .rb files that similarly require 'shellwords', but do not declare it in the gemspec.

File: lib/gym/runner.rb (hunk 2 above)
Comment (mpirri) [inline lib/gym/runner.rb]: Yes - great catch! I will change this.

File: lib/gym/runner.rb (hunk 1 above)
Comment (KrauseFx) [inline lib/gym/runner.rb]: Good call, AFAIK it's in the standard library and doesn't need to be added. We use it already in `gym`

**Comment by KrauseFx** [comment]
Looks good to me, test succeed (just one job got stuck), ready to :ship:

**Pull request merged**